frontend/src/pages/GeneralSettings/EmbeddingPreference/index.jsx

```diff
@@ -97,6 +99,15 @@ const EMBEDDERS = [
     options: (settings) => <LiteLLMOptions settings={settings} />,
     description: "Run powerful embedding models from LiteLLM.",
   },
+  {
+    name: "Generic OpenAI",
+    value: "generic-openai",
+    logo: GenericOpenAiLogo,
+    options: (settings) => (
+      <GenericOpenAiEmbeddingOptions settings={settings} />
+    ),
+    description: "Run powerful embedding models from OpenAI.",
```

Comment:
This is the generic provider. This description should be something like "Run embedding models from any OpenAI compatible API service."